packages/isomorphic/src/utils/browser.ts

```diff
@@ -13,5 +12,24 @@ export const bytesToHex: typeof BytesToHexFn = (bytes) => {
   return hex.toUpperCase()
 }
 
-export const hexToBytes: typeof HexToBytesFn = nobleHexToBytes
+// A clone of hexToBytes from @noble/hashes without the length checks. This allows us to do our own checks.
+/* eslint-disable-next-line func-style -- Typed to ensure uniformity between node and browser implementations and docs */
```

Comment:
nit: This comment doesn't quite make sense to me considering the es-lint error is for using the `const hexToBytes = (hex) => {...}` syntax for the function instead of `function hexToBytes(hex) {...}`, but the comment is talking about typing.

Reply:
This is a complaint about `typeof HexToBytesFn` which is needed to ensure uniformity.

Reply:
Moved to a wrapping enable/disable since the two additional methods I added need it too.

Addressed in 11dd21c101084f6c00ee9d606bbdb0ad1dc048e7